variables.tf

```diff
@@ -30,3 +32,70 @@ variable "scan_on_push" {
   description = "Scan image on push to repo."
   default     = true
 }
+
+# ECS variables
+
+variable "app_name" {
+  type        = string
+  description = "Name of the application"
+}
+
+variable "environment" {
+  type        = string
+  description = "Environment name"
+}
+
+variable "task_name" {
+  type        = string
+  description = "Name of the task to be run"
+}
+
+variable "ecs_vpc_id" {
+  type = string
+  description = "VPC ID to be used by ECS"
+}
+
+variable "ecs_subnet_ids" {
+  description = "Subnet IDs for the ECS tasks."
+  type        = list(string)
+}
+
+variable "repo_tag" {
+  type        = string
+  description = "The tag to identify and pull the image in ECR repo"
+  default     = "latest"
+}
```

Comment:
These variables that are prefixed with `repo` need cleaned up a bit to provide more context. Which belong to ECR, and which to GitHub?

Reply:
updating to ecr_repo_tag